app/components/primer/button_group_component.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use ButtonGroupComponent to render a series of buttons.
+  class ButtonGroupComponent < Primer::Component
+    include ViewComponent::SlotableV2
+
+    renders_many :buttons, ->(classes: "", **kwargs) do
```

Comment:
What do you think is the best way to document that these slots take all of the arguments for ButtonComponent?

Reply:
That's a good question. I don't think we aren't documenting slots in other components
@srt32 wdyt?

Reply:
I've seen us document slots v1 behavior like this https://github.com/primer/view_components/pull/182/files#diff-a870f147518fb3db01d7aacb41bc479052f20a68f7e0f934fc78715d1851dd68R36-R39. We'd have to rewrite the Rakefile parser to handle slots v2?

Reply:
Continued over in https://github.com/primer/view_components/pull/185#discussion_r569558545